wiso.js

```diff
@@ -36,19 +36,21 @@
 
 
 function detectWeb(doc, url) {
-	if (url.search(/START=A[246]0/) !== -1) {
+	if (url.search(/document/) !== -1) {
 		if( ZU.xpath(doc, '//a[contains(@class, "boxExport")]').length>0 ) {
 			//single item --> generic fallback = journalArticle
 			return "journalArticle";
 		}
-	} else if ( ZU.xpath(doc, '//a/span[contains(@class, "boxHeader")]').length>0 ) {
+	} else if (getSearchResults(doc, true)) {
 		return "multiple";
 	}
 }
 
 
 function scrape(doc, url) {
-	var risUrl = url+"&OHNE_SAVE=1&T_TEMPLATE=ris";
+	var permaUrl = ZU.xpathText(doc, '//pre[strong]/text()');
```

Comment:
That is a restriction on the text node, because there is also a `strong`-node, e.g.
```
<pre class="text">
   <strong>Dauerhafte Adresse des Dokuments: </strong>
   https://www.wiso-net.de/document/BEFO__20071105986-E-FIZT-BEFO-DOMA-ZDEE-ETEC
</pre>
```
BTW do you think the xpath is specific enough? Or should I filter on the text to be an url also?